src/AbstractProxyFixer.php

```diff
@@ -53,6 +61,22 @@ public function getPriority()
     /**
      * {@inheritdoc}
      */
+    public function supports(\SplFileInfo $file)
+    {
+        return $this->proxyFixer->supports($file);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setWhitespacesConfig(WhitespacesFixerConfig $config)
+    {
+        return $this->proxyFixer->setWhitespacesConfig($config);
```

Comment:
Thought it but was missing the right way to implement it.
Now found: I'll call the parent and the proxyfixer, so the config is available for both, and the check is done for both.